app/validators/digital_object/dynamic_fields_validator.rb

```diff
@@ -51,6 +42,25 @@ def field_errors(configuration, value)
       send("errors_for_#{configuration[:field_type]}_field", configuration, value)
     end
 
+    def errors_for_field_group(value, field_or_group_key, new_path, field_map_subtree)
+      errors = []
+      if value.is_a?(Array)
+        errors.append([new_path, "is not repeatable"]) if violates_repeat_value_constraint(field_or_group_key, value)
+
+        value.each_with_index do |v, i|
+          errors.concat errors_for(field_map_subtree[:children], v, "#{new_path}[#{i}]")
+        end
+      else
+        errors.append([new_path, "must contain an array"])
+      end
+      errors
+    end
+
+    def violates_repeat_value_constraint(field_or_group_key, value)
```

Comment:
```suggestion
    def violates_repeat_value_constraint(group_key, value)
```
Recommending change because this method only expects a field group and wouldn't be receiving a field.